_assets/ci/Jenkinsfile

```diff
@@ -36,6 +36,7 @@ pipeline {
     }
     stage('Setup') { steps { dir(env.STATUS_PATH) {
       sh 'make setup'
+      sh 'make xgo'
```

Comment:
I guess a better name would be `make install-xgo` to keep in line with existing targets?